src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorSlaveIntegrationTest.groovy

```diff
@@ -59,12 +58,35 @@ class IqPolicyEvaluatorSlaveIntegrationTest
     WireMock.configureFor("localhost", wireMockRule.port())
     givenThat(get(urlMatching('/rest/config/proprietary\\?.*'))
         .willReturn(okJson('{}')))
-    givenThat(put(urlMatching('/rest/ci/scan/.*'))
-        .willReturn(okJson('{"scanId":"scanId"}')))
-    givenThat(post(urlMatching('/rest/policy/.*'))
-        .willReturn(okJson('{}')))
     givenThat(post(urlMatching('/rest/integration/applications/verifyOrCreate/.*'))
         .willReturn(okJson('true')))
+    givenThat(post(urlMatching('/rest/integration/applications/app/evaluations/ci/stages/.*'))
+        .willReturn(okJson('{"statusId": "statusId"}')))
+    givenThat(get(urlMatching('/rest/integration/applications/app/evaluations/status/statusId'))
+        .willReturn(okJson('''{
```

Comment:
it just occurred to me that the old code just used `'{}'`, so I'm wondering if all we need to put in there is the `status`, `scanReceipt`, and `result` properties with empty objects of `{}`.  Worth a try to see what happens.  I think the scanReceipt just needs a scanId, and that would be about it.

Reply:
> so I'm wondering if all we need to put in there is the `status` with statusId, `scanReceipt` with scanId, and `result` with empty `{}`

The minimum we need currently is the completed status, reason, scanReceipt with scanId and reportUrl.  Otherwise, there are failures in the nexus-platform-api code.  The fact that these tests are passing values through the nexus-platform-api code, I would suggest we keep the payload valid vs just retaining values that don't currently cause a failure.